src/Random.cpp

```diff
@@ -85,13 +88,43 @@ namespace galsim {
         clearCache();
     }
 
+    // Next two functions shamelessly stolen from
+    // http://stackoverflow.com/questions/236129/split-a-string-in-c
+    std::vector<std::string>& split(const std::string& s, char delim,
+                                    std::vector<std::string>& elems) {
+        std::stringstream ss(s);
+        std::string item;
+        while (std::getline(ss, item, delim)) {
+            elems.push_back(item);
+        }
+        return elems;
+    }
+
+    std::vector<std::string> split(const std::string& s, char delim) {
+        std::vector<std::string> elems;
+        split(s, delim, elems);
+        return elems;
+    }
+
+    std::string seedstring(std::vector<std::string> seed) {
```

Comment:
Just noticed that this is passed by value.  The argument should be a const reference to avoid the gratuitous copy.